sys/shell/commands/sc_gnrc_icmpv6_echo.c

```diff
@@ -340,8 +376,9 @@ static void _print_reply(_ping_data_t *data, gnrc_pktsnip_t *icmpv6,
         }
         recv_seq = byteorder_ntohs(icmpv6_hdr->seq);
         ipv6_addr_to_str(&from_str[0], from, sizeof(from_str));
+
+        /* we can only calculate RTT if payload was large enough for TX timestamp */
         if (data->datalen >= sizeof(uint32_t)) {
-            triptime = xtimer_now_usec() - unaligned_get_u32(icmpv6_hdr + 1);
```

Comment:
How is trip-time set now for the checks below? From what I can tell it is only set in `_check_payload()` which is called _after_ these lines. At least for the first message the RTT will now be wrong.

Reply:
Ops, forgot to move this when I shuffled the error printing around